src/Symfony/Component/FeatureFlags/README.md

```diff
@@ -0,0 +1,77 @@
+FeatureFlags Component
+======================
+
+The FeatureFlags component provides TODO
+
+**This Component is experimental**.
+[Experimental features](https://symfony.com/doc/current/contributing/code/experimental.html)
+are not covered by Symfony's
+[Backward Compatibility Promise](https://symfony.com/doc/current/contributing/code/bc.html).
+
+Usage
+-----
+
+This example implements the following logic :
+* **Grants** when the `feat` parameter equals to `true` in the query string.
+* **Denies** when the `feat` parameter equals to `false` in the query string.
+* **Abstain** when the `feat` parameter is not found in the query string. So it
+ fallbacks to `false`.
+
+```php
+<?php
+
+use Symfony\Component\FeatureFlags\Feature;
+use Symfony\Component\FeatureFlags\FeatureChecker;
+use Symfony\Component\FeatureFlags\FeatureCollection;
+use Symfony\Component\FeatureFlags\Strategy\RequestQueryStrategy;
+
+$features = FeatureCollection::withFeatures([
+    new Feature(
+        name: 'new_feature',
+        description: 'My new feature',
+        default: false,
+        strategy: new RequestQueryStrategy('feat'),
+    ),
+];
+$featureChecker = new FeatureChecker(
+    features: $features,
+    whenNotFound: false,
+);
+
+if ($featureChecker->isEnabled('new_feature')) {
+    // Use the new feature
+} else {
+    // Use the legacy code
+}
+```
+
+Available strategies
```

Comment:
as per https://github.com/symfony/symfony/pull/51649#issuecomment-1748543083
an abstract strategy or a way to plug a "simple class" could be valuable :)

Reply:
Implement the StrategyInterface and with autowire you're good to go.

Reply:
yes I got it already, it is to avoid having to deal with internal component "strategy result" but more a simple boolean (flag on|off) from the ppl dev code

like for example the security voter does
the code+attribute on the comment mentioned would be very useful if plugged into the strategy logic :)

Reply:
Ok got it. I misunderstood you're point. It could indeed be something added. I'll have to think about it.